.gitignore

```diff
@@ -59,3 +59,5 @@ UpgradeLog*.htm*
 packages/
 GitExtensions.*.sln.VisualState.xml
 GitExtensions.settings.backup
+Bin/Git-1.9.5-preview20141217.exe
```

Comment:
Yeah, I thought so but then I scared to have too broad filter. Now I am not scared anymore and put this in ;)